### DISCUSSION TITLE
heavy model by 100-500MB its real?

### DISCUSSION THREAD

**Discussion by shurshilov** [vladmandic/human#415] [Ideas]
What if we make some very heavy but more accurate model? for kiosk modes, when system administrators download at least 100-500 megabytes of neural network once, but they will do it only once, but the accuracy, as I understand it, will be higher?

**Comment by vladmandic** [comment]
what is the exact use-case? if its to run in browser, then no as running inside browser always go through more layers than necessary, either wegbl or wasm or webgpu and you're limited to how big of a model you're allowed to run in browser to start with. if its to run in backend, then yes.

accuracy of what?

but in general, that is not the purpose of this library.

**Reply by shurshilov** [reply]
very often, customers use kiosk mode. This is usually a tablet that is turned on 24 by 7, and when a person approaches, he recognizes it and takes it off at work. It also has 2 modes, 1 mode when the camera recognizes the person itself, the second is when the person first chooses himself. The second option is of course much more accurate because in fact a person is compared only with his photo and not with a sample. It's just that in kiosk mode there are almost no restrictions on the size of the model, the browser is used simply because it is used as a rule in erp, which also have a web interface.
I just realized that it is extremely difficult to achieve accurate photo recognition. The same iPhone uses infrared sensors and face volume sensors - of course, this is almost 100% accurate. 
What if we created a descriptor not from a photo, but from a video in which a person turns his head left and right, thereby creating a "volume" and further, when recognizing employees, also turning their heads is not a problem for most campaigns...

**Reply by shurshilov** [reply]
And of course, thank you, I have sold more than 40 copies of my software on your library))

**Reply by vladmandic** [reply]
if it runs on tablet, it 99% chance means its running browser. and then you're limited with tablets hardware as well as browser's limitations - neither of those are well suited to run complicated models.

but regarding using multiple embeddings from video? absolutely. i've done that in the past.
